paz/pipelines/keypoints.py

```diff
@@ -14,6 +14,11 @@
 from ..backend.keypoints import flip_keypoints_left_right, uv_to_vu
 from ..datasets import JOINT_CONFIG, FLIP_CONFIG
 
+import numpy as np
```

Comment:
please put this on the top. Often we have standard library imports on top. Then common library imports second and then our library specific imports third.